lib/hypgen/workflow.rb

```diff
@@ -53,6 +53,8 @@ def run!
         output = pipe.read
 
         puts "Workflow execution output: #{output}"
+        pipe.close
+        $?.to_i
```

Comment:
@mpawlik why do you ask me? This is [your](https://github.com/ismop/hypgen/blame/experiment_completion/lib/hypgen/workflow.rb) change :smile_cat: